processor/elasticprocessor/internal/hostmetrics/cpu.go

```diff
@@ -5,7 +5,7 @@ import (
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
-func addCPUMetrics(metrics pmetric.MetricSlice, dataset string) error {
+func addCPUMetrics(metrics pmetric.MetricSlice, rm pcommon.Resource, dataset string) error {
```

Comment:
nit: can we just call this `resource`, so it's more obvious what it is when it's used later?

Reply:
Done